pageserver/src/tenant/timeline.rs

```diff
@@ -1060,8 +1060,7 @@ impl Timeline {
 
         // start counting after throttle so that throttle time
         // is always less than observation time
-        let throttled = self
-            .timeline_get_throttle
+        self.timeline_get_throttle
```

Comment:
The comment above this line is obsolete, no?

Reply:
be2d64bee396b262ae187bf04c427fd24065eaf4